### DISCUSSION TITLE
File paths in list view can be shortened to save space

### DISCUSSION THREAD

**Discussion by pluma9** [ray-x/navigator.lua#251] [Ideas]
I have a project that has a deep file hierarchy. Roughly 1/3 of the file path in the list view is the path to my project's root folder. Do you plan to have any option that enables relative path mode?

**Comment by ray-x** [comment]
It has been relative path

![image](https://user-images.githubusercontent.com/1681295/209476327-74c6f562-22dd-424e-a2c5-9fd8a44bb7a6.png)

Do you mean you need further shorten the path?
Could you give a example?

**Reply by pluma9** [reply]
The path in the green rectangle is the path leading to the project root directory. This happens to a C# project. My NodeJS typescript project does use relative paths.

![image](https://user-images.githubusercontent.com/16544647/209500622-dc44a17b-3244-4e55-b1ba-44fca83c7dd1.png)

**Comment by ray-x** [comment]
Ok, it might be a windows specific or C# specific issue.
